airflow/providers/google/marketing_platform/hooks/display_video.py

```diff
@@ -151,3 +167,73 @@ def upload_line_items(self, line_items: Any) -> List[Dict[str, Any]]:
             .execute(num_retries=self.num_retries)
         )
         return response
+
+    def create_sdf_download_operation(self, body_request: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Creates an SDF Download Task and Returns an Operation.
+        :param body_request: Body request.
+        :type body_request: Dict[str, Any]
+
+        More information about body request n be found here:
+        https://developers.google.com/display-video/api/reference/rest/v1/sdfdownloadtasks/create
+        """
+
+        result = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .sdfdownloadtasks()
+            .create(body=body_request)
+            .execute(num_retries=self.num_retries)
+        )
+        return result["name"]
+
+    def get_sdf_download_operation(self, name: Dict[str, Any]):
+        """
+        Gets the latest state of an asynchronous SDF download task operation.
+        :param name: The name of the operation resource.
+        :type name: str
+        """
+
+        result = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .sdfdownloadtasks()
+            .operation()
+            .get(name=name)
+            .execute(num_retries=self.num_retries)
+        )
+
+        return result
+
+    def download_media(self, resource_name: str):
+        """
+        Downloads media.
+        :param resource_name:  of the media that is being downloaded.
+        :type resource_name: str
+        """
+
+        request = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .media()
+            .download_media(resource_name=resource_name)
+        )
+        return request
+
+    def wait_for_download_operation(self, operation: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Suggest polling operation be performed in a sensor.  See comments on GoogleDisplayVideo360SDFtoGCSOperator for more details.

Reply:
Done.